src/index.js

```diff
@@ -160,7 +163,8 @@ export default class ScrollContainer extends PureComponent {
   }
 
   onMouseDown = (e) => {
-    if (this.isDraggable(e.target)) {
+    const { ignoredMouseButtons } = this.props
+    if (this.isDraggable(e.target) && !ignoredMouseButtons.includes(e.button)) {
```

Comment:
It's better to not use `includes` in the code of library, because it will require polyfills for IE and some other old browsers.